app/components/Audio/Player.js

```diff
@@ -5,33 +5,31 @@ import {
 } from 'react-native';
 import Controls from './Controls';
 import Video from 'react-native-video';
-import {connect} from 'react-redux'
-import {fetchAudioUrl} from '../../store/action'
+import { connect } from 'react-redux'
+import { fetchAudioUrl } from '../../store/action'
 import Color from '../../utils/colorConstants'
-// import {styles} from './styles'
 
 class Player extends Component {
   constructor(props) {
     super(props);
-    console.log("PLAYER ",props)
+    console.log("PLAYER ", props)
```

Comment:
console.log is needed here?

Reply:
It is not needed, as I have  removed console and unwanted comments or file manually so  may be at few places forgot to remove . In next commit I'll remove it .